v2/jwk/jwk.go

```diff
@@ -105,3 +124,19 @@ func RefreshKeySets() (err error) {
 
 	return
 }
+
+func getKeySetsURL() (string, error) {
+	if config == nil && KeySetURL == "" {
+		return "", errors.New("jwk is not configured")
+	}
+
+	if config == nil {
+		return KeySetURL, nil
+	}
+
+	if config.Stage == stages.StageProd {
+		return "https://sso-api.users.enlight.skf.com/jwks", nil
+	}
+
+	return "https://sso-api." + config.Stage + ".users.enlight.skf.com/jwks", nil
```

Comment:
Why not sprintf here?

Reply:
I would only use `Sprintf` when it would add to readability or make life simpler, like when you have multiple variables or the variables are not only strings.

I don't believe that is the case here.